cordyceps/src/lib.rs

```diff
@@ -59,6 +62,15 @@
 //!   A [`TransferStack`] can be used to efficiently transfer ownership of
 //!   resources from multiple producers to a consumer, such as for reuse or
 //!   cleanup.
+//!
+//!   This structure is only available if the target supports CAS (Compare and
+//!   Swap) atomics.
+//!
+//! ## Compatibility
+//!
+//! Rudimentary support for targets without CAS (Compare and Swap) atomics, such as
+//! Cortex-M0+/`thumbv6m-none-eabi`, is provided, however not all structures and
+//! features may be available.
```

Comment:
oh, one more little nit: i might move this block out of the lib.rs docs, and put it in the [README](https://github.com/hawkw/mycelium/blob/55181917e65b619f756c6d12c26c999d7c8d6646/cordyceps/README.md), which is included at the top of the lib.rs docs here: https://github.com/hawkw/mycelium/blob/55181917e65b619f756c6d12c26c999d7c8d6646/cordyceps/src/lib.rs#L1

the list of individual data structures is in the lib.rs docs rather than in the README that gets included into it because it contains intra-docs links to the different types. but the compatibility section doesn't include any intra-docs links, so it might be nicer to have it in the portion that's also in the README and gets rendered on crates.io etc.